graphprot/NeuralNet.py

```diff
@@ -154,27 +161,27 @@ def plot_acc(self):
             plt.savefig('acc_epoch.png')
             plt.close()
 
-
     def plot_hit_rate(self, data='eval', threshold=4, mode='percentage'):
-        '''
-        Plots the hitrate as a function of the models' rank
-        
-        - threshold (default = 4): defines the value to split into a hit (1) or a non-hit (0)
-        - mode : displays the hitrate as a number of hits ('count') or as a percentage ('percantage')
-        '''
+        """Plots the hitrate as a function of the models' rank
+
+        Args:
+            data (str, optional): which stage to consider train/eval/test. Defaults to 'eval'.
+            threshold (int, optional): defines the value to split into a hit (1) or a non-hit (0). Defaults to 4.
+            mode (str, optional): displays the hitrate as a number of hits ('count') or as a percentage ('percantage') . Defaults to 'percentage'.
+        """
 
         import matplotlib.pyplot as plt
-            
-        try :
+
+        try:
 
             hitrate = self.get_metrics(data, threshold).HitRate()
-            
```

Comment:
```
        hitrate = list(self.get_metrics(data, threshold).HitRate())

        nb_models = len(hitrate)
        X = range(1, nb_models + 1)

        if mode == 'percentage' :
            tot_hit = hitrate.count(1)
            hitrate = [x/tot_hit for x in hitrate]

```

Reply:
The hitrate should be the number of hits at rank X divided by the total number of hits.

Reply:
yes it should :) I'll fix that

Reply:
My bad, I wrote that !:D

Reply:
plus hitrate is a np array so you can divide it directly by a scalar (or an array of same size), i.e. `hitrate /= hitrate.sum()`